client/src/controllers/TabsController.ts

```diff
@@ -0,0 +1,319 @@
+/* eslint-disable no-shadow */
```

Comment:
```suggestion
```

Once you rebase on the latest `main` you should not need this, we have fixed this linting issue.